src/main/scala/dpla/ingestion3/mappers/providers/NaraMapping.scala

```diff
@@ -85,36 +86,40 @@ class NaraMapping extends XmlMapping with XmlExtractor {
-        case (Some("Restricted - Possibly"), Some("Service Mark")) => Some(URI("http://rightsstatements.org/vocab/NoC-OKLR/1.0/"))
-        case (Some("Restricted - Possibly"), Some("Trademark")) => Some(URI("http://rightsstatements.org/vocab/NoC-OKLR/1.0/"))
-        case (Some("Restricted - Possibly"), Some("Other")) => Some(URI("http://rightsstatements.org/vocab/NoC-OKLR/1.0/"))
-        case (Some("Undetermined"), _) => Some(URI("http://rightsstatements.org/vocab/UND/1.0/"))
-        case (Some("Unrestricted"), _) => Some(URI("http://rightsstatements.org/vocab/NoC-US/1.0/"))
+        case (Some("Restricted - Fully"), Some("Copyright")) => Some(rightsUri("InC"))
+        case (Some("Restricted - Fully"), Some("Donor Restrictions")) => Some(rightsUri("InC"))
+        case (Some("Restricted - Fully"), Some("Public Law 101-246")) => Some(rightsUri("InC"))
+        case (Some("Restricted - Fully"), Some("Service Mark")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Fully"), Some("Trademark")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Fully"), Some("Other")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Partly"), Some("Copyright")) => Some(rightsUri("InC"))
+        case (Some("Restricted - Partly"), Some("Donor Restrictions")) => Some(rightsUri("InC"))
+        case (Some("Restricted - Partly"), Some("Public Law 101-246")) => Some(rightsUri("InC"))
+        case (Some("Restricted - Partly"), Some("Service Mark")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Partly"), Some("Trademark")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Partly"), Some("Other")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Possibly"), Some("Copyright")) => Some(rightsUri("UND"))
+        case (Some("Restricted - Possibly"), Some("Donor Restrictions")) => Some(rightsUri("UND"))
+        case (Some("Restricted - Possibly"), Some("Public Law 101-246")) => Some(rightsUri("NKC"))
+        case (Some("Restricted - Possibly"), Some("Service Mark")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Possibly"), Some("Trademark")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Possibly"), Some("Other")) => Some(rightsUri("NoC-OKLR"))
+        case (Some("Restricted - Possibly"), _) => Some(rightsUri("UND"))
+        case (Some("Undetermined"), _) => Some(rightsUri("CNE"))
+        case (Some("Unrestricted"), _) => Some(rightsUri("NoC-US"))
         case (_, _) => None
       }}
 
     // The most restrictive statements
     val mostRestrictive = Seq(
-      Option(URI("http://rightsstatements.org/vocab/InC/1.0/")), // in copyright
-      Option(URI("http://rightsstatements.org/vocab/CNE/1.0/")) // copyright not evaluated
+      Option(rightsUri("InC")), // in copyright
+      Option(rightsUri("CNE"))  // copyright not evaluated
```

Comment:
I think this "CNE" should have been switched to "UND" as well?

Reply:
Honestly I don't know -- just following the spec here. If you say so I can change it

Reply:
Yes, I am just suggesting that all CNE/UND should have been  switched in the file, since their meanings were exactly reversed originally. Including here, and also in  the commented section at line 139 & 141. CNE is never going to be part of the "most restrictive" default conversation any more, because it doesn't map to any of the specific use restrictions anymore, only UND does.